app/views/stats/index.html.erb

```diff
@@ -24,12 +24,20 @@
 <div class="wrapper">
   <dl class="statistics meta group" role="complementary">
     <% [:user_subscriptions, :kudos, :comments, :bookmarks, :subscriptions, :word_count].each do |stat| %>
-      <dt><%= (stat.to_s.titleize == "Comments") ? ts("Comment Threads") : stat.to_s.titleize %></dt>
-      <dd><%= @totals[stat] %></dd>
+      <% if stat.to_s.titleize == "User Subscriptions"
+          html_class = 'user subscriptions'
```

Comment:
If it's intentional that `html_class` in this case becomes two classes, the conditional could be simplified to have only one exception branch (the Word Count case), and the rest to assign `html_class = stat.to_s.humanize.downcase` (`humanize` gets rid of any underscores, but upcases the first letter, so `downcase` to lowercase everything).

And I know this is pre-existent code, but I can't think of a reason why the whole comparison couldn't be done on symbols to begin with. By which I mean:
```
<% if stat == :word_count
    html_class = 'words'
  else
    html_class = stat.to_s.humanize.downcase
  end
%>
<dt class="<%= html_class %>"><%= (stat == :comments) ? ts("Comment Threads") : stat.to_s.titleize %></dt>
```